homeassistant/brands/leviosa.json

```diff
@@ -0,0 +1,5 @@
+{
+  "domain": "leviosa",
+  "name": "Leviosa",
+  "integrations": ["leviosa_shades"]
+}
```

Comment:
Let's add this to a follow-up PR as well. This will also require a logo for `leviosa` domain. 

And since it seems Leviosa only offers one type of services, the Leviosa Shades, I am not sure if adding the brand adds any value here. See https://developers.home-assistant.io/docs/creating_integration_brand/.

Reply:
Suggestion accepted, thanks.